docs/source/pika-comparison.rst

```diff
@@ -0,0 +1,288 @@
+
+.. _Pika: https://pika.readthedocs.io/en/stable/
+.. _RabbitMQ tutorials: https://www.rabbitmq.com/getstarted.html
+.. _work queues: https://www.rabbitmq.com/tutorials/tutorial-two-python.html
+.. _topics: https://www.rabbitmq.com/tutorials/tutorial-five-python.html
+.. _RPC: https://www.rabbitmq.com/tutorials/tutorial-six-python.html
+.. _aio-pika: https://aio-pika.readthedocs.io/en/latest/index.html
+
+Coming from Pika
+================
+
+KiwiPy comes natively with a RabbitMQ communicator (others can be added by extending the :py:class:`~kiwipy.Communicator` interface) and thus it may be useful to see how to achieve the same things in `Pika`_ (the standard library used in the `RabbitMQ tutorials`_) and kiwiPy.  This also shows some of the differences, particularly where kiwiPy is less verbose by setting sensible defaults for the user.
+
+
+Work Queues
+-----------
+
+Let's start with RabbitMQ's `work queues`_ example.
+
+Pika
+++++
+
+The code for sending a task::
+
+    import pika
+    import sys
+
+    connection = pika.BlockingConnection(
+        pika.ConnectionParameters(host='localhost'))
+    channel = connection.channel()
+
+    channel.queue_declare(queue='task_queue', durable=True)
+
+    message = ' '.join(sys.argv[1:]) or "Hello World!"
+    channel.basic_publish(
+        exchange='',
+        routing_key='task_queue',
+        body=message,
+        properties=pika.BasicProperties(
+            delivery_mode=2,  # make message persistent
+        ))
+    print(" [x] Sent %r" % message)
+    connection.close()
+
+The code for running a worker::
+
+    import pika
+    import time
+
+    connection = pika.BlockingConnection(
+        pika.ConnectionParameters(host='localhost'))
+    channel = connection.channel()
+
+    channel.queue_declare(queue='task_queue', durable=True)
+    print(' [*] Waiting for messages. To exit press CTRL+C')
+
+
+    def callback(ch, method, properties, body):
+        print(" [x] Received %r" % body)
+        time.sleep(body.count(b'.'))
+        print(" [x] Done")
+        ch.basic_ack(delivery_tag=method.delivery_tag)
+
+
+    channel.basic_qos(prefetch_count=1)
+    channel.basic_consume(queue='task_queue', on_message_callback=callback)
+
+    channel.start_consuming()
+
+
+KiwiPy
+++++++
+
+And now, in kiwiPy.  The code for sending the same task:
+
+.. literalinclude:: examples/pika/new_task.py
+
+Here, compared to the Pika snippet, we see that we don't have to decide or know about an ``exchange``, ``routing_key``, ``properties``, ``channel`` or durability of the message.  Tasks queues in kiwiPy are always durable and the corresponding messages persistent.  The ``routing_key`` is simply the name of the queue that the user declared and an ``exchange`` is selected automatically for the user (set to ``kiwipy.tasks`` by default).
```

Comment:
Do you want to say here if it is possible to change these defaults should the user care to? Or does the API not allow this?

Reply:
Added a comment explaining how to change defaults.